src/Native/System.Native/pal_process.h

```diff
@@ -5,6 +5,9 @@
 
 #include "pal_types.h"
 #include <string.h>
+#include <sys/param.h>
+
+struct rusage_info_v3;
```

Comment:
I don't understand why we have a forward declaration to something we only cast to something else.